### DISCUSSION TITLE
Excluding infrastructure from calculations

### DISCUSSION THREAD

**Discussion by JuliSant** [LCA-ActivityBrowser/activity-browser#949] [Q&A]
Hi!

I know that in SimaPro and OpenLCA you can choose to exclude infrastructure in background processes from calculations. Is there also a way of doing that in AB?

Thanks in advance!

Best
Julia

**Comment by CriseldaCaddiganQ5r** [comment]
AB does not have a built-in feature for excluding infrastructure in background processes from calculations. However, you can achieve a similar result by creating a separate process flow that represents the infrastructure and running it in parallel with the main process flow.
